collections-lists/RandomizeGenericListsInCSharp/RandomizeGenericListsTests/RandomizeGenericListsUnitTests.cs

```diff
@@ -0,0 +1,63 @@
+using RandomizeGenericListsInCSharp;
+
+namespace RandomizeGenericListsTests;
+
+[TestClass]
+public class RandomizeGenericListsUnitTests
+{
+    private readonly RandomizeGenericListsMethods _randomizeListObj;
+    private readonly List<int> _orderedList;
+
+    public RandomizeGenericListsUnitTests()
+    {
+        _randomizeListObj = new RandomizeGenericListsMethods();
+        _orderedList = _randomizeListObj.OrderedInts(1000000);
+    }
+
+    [TestMethod]
+    public void GivenAnOrderedList_WhenNotEmpty_VerifyShuffleGuid()
+    {
+        var shuffledList = _randomizeListObj.GenerateRandomOrderByGuid(_orderedList);
+
+        var firstVal = shuffledList.First();
+        var lastVal = shuffledList.Last();
+        
+        Assert.IsFalse(firstVal == 0);
```

Comment:
Instead of `Assert.IsTrue()`, we should be using `Assert.AreEqual()` when comparing values and not explicitly checking the value of a `bool`.